### ISSUE TITLE
Add additional URL for ReadMe

### ISSUE THREAD

**Issue by kanadgupta** [2factorauth/twofactorauth#5094]
- [x] I have searched [open issues and pull requests](https://github.com/2factorauth/twofactorauth/issues?q=is%3Aopen). The issue I'm creating is not a duplicate of an existing open issue or pull request.

### Information about the site entry to be updated: ###

* Site name - ReadMe

* Site URL - https://readme.io

* Site category - Developer

### Information about why the site entry should be updated: ###
**This site entry should be updated because:**
- [ ] It now supports 2FA.
- [ ] The methods of 2FA that it supports have changed.
- [ ] It has begun the process of implementing 2FA.
- [ ] It no longer supports 2FA.
- [ ] Its name has changed.
- [ ] Its documentation link has changed.
- [ ] Its Twitter username, Facebook page, and/or email address have changed.
- [ ] It fits better in a different category.
- [ ] Its logo has changed.
- [ ] It has been acquired or absorbed by another site, and the new site is eligible to be listed on twofactorauth.org.
- [x] Other (please describe below).

### Information about how the site entry should be updated: ###

We still own the original [readme.io](https://readme.io/) domain, but in the last year we have acquired [readme.com](https://readme.com) and have been slowly rolling over our services to that domain. We recently updated our login page at [dash.readme.io](https://dash.readme.io/) to redirect to [dash.readme.com](https://dash.readme.com/) (which you can verify by clicking the link). Is there a way we can somehow add an additional URL to our entry so password managers like 1Password can detect both URLs? If not, I'll just open a PR to update the URL on this entry to [readme.com](https://readme.com). Thanks so much!

Original PR for reference: https://github.com/2factorauth/twofactorauth/pull/4019

**Comment by Carlgo11** [comment]
Hi @kanadgupta,
no sadly it's not possible (yet) to link multiple domains. 1Password might be able to do it manually for you but I've understood that recently a few other password managers have started to use our data as well and contacting each one might be a bit tricky 😅

**Comment by phallobst** [comment]
This question is related to #5072. An "alias" section (or something similar) could solve the problem with moving to a new TLD as well.

**Comment by Carlgo11** [comment]
@phallobst I already have a YAML schema with aliases implemented but I'm waiting on making a PR until the other YAML stuff have been merged
